### PR TITLE
log rank test/chi-squared test

### PR DESCRIPTION
**Pull request by hideaki**
### Description
Add support for log rank test/chi-squared test

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [x] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/survival.R [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  grouped_col <- grouped_by(df)
+
+  if (!is.null(substitute(cohort))) {
+    cohort_col <- col_name(substitute(cohort))
+    if (!is.null(cohort_func)) {
+      df[[cohort_col]] <- extract_from_date(df[[cohort_col]], type=cohort_func)
+    }
+  }
+  else {
+    cohort_col <- "1"
```
Comment (kei51e) [inline R/survival.R]: What is this "1" for?

File: R/survival.R [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  }
+
+  # substitute is needed because time can be
+  # NSE column name and it throws an evaluation error
```
Comment (kei51e) [inline R/survival.R]: What is NSE?

Comment (hideaki) [reply R/survival.R]: It stands for Non-Standard Evaluation. Added comment.

File: R/survival.R (hunk 1 above)
Comment (hideaki) [reply R/survival.R]: Added comment on it.

**Pull request merged**